aioWiserHeatAPI/heating_actuator.py

```diff
@@ -172,52 +173,6 @@ def output_type(self) -> str:
         """Get output type"""
         return self._device_type_data.get("OutputType", TEXT_UNKNOWN)
 
-    # added by LGO
```

Comment:
What is the reason for removing all these?  Aren't they needed for the v1 hub as it doesn't have the equipment dataset?